lib/galaxy/jobs/runners/drmaa.py

```diff
@@ -247,7 +247,7 @@ def check_watched_items( self ):
                 assert external_job_id not in ( None, 'None' ), '(%s/%s) Invalid job id' % ( galaxy_id_tag, external_job_id )
                 state = self.ds.job_status( external_job_id )
             except ( drmaa.InternalException, drmaa.InvalidJobException ) as e:
-                if isinstance( e , drmaa.InvalidJobException ):
+                if isinstance( e, drmaa.InvalidJobException ):
```

Comment:
Maybe not across the board.  I've started making a pass fixing a few, but most of them are triggered by ```:``` dictionary definitions, which we don't want to mangle the formatting of.